docs/gateway/nas.md

```diff
@@ -109,6 +109,6 @@ NAS gateway implementation allows symlinks on regular files.
 
```

Comment:
Same note w.r.t. this being deprecated, we can probably safely drop all of the /gateway/* docs

Reply:
Leaving the docs for existing references.
But I added a note about Gateway's deprecation with the link to the blog post on the readme.